index.html

```diff
@@ -29,12 +29,12 @@
                 window.onload = function() {
                     document.body.innerHTML
                         = "The application failed to load, missing file: "
-                            + fileRef;
+                        + fileRef;
```

Comment:
I disagree with the indentation change introduced here, please remove it from this PR.